.github/workflows/lablink-images.yml

```diff
@@ -385,3 +385,35 @@ jobs:
           docker run --rm ${{ steps.get_tag.outputs.image }} bash -c 'source /home/client/.venv/bin/activate && timeout 2 check_gpu > /tmp/check_gpu_output.txt 2>&1 || true && test -s /tmp/check_gpu_output.txt && echo "check_gpu entry point executed"'
           docker run --rm ${{ steps.get_tag.outputs.image }} bash -c 'source /home/client/.venv/bin/activate && timeout 2 subscribe > /tmp/subscribe_output.txt 2>&1 || true && test -s /tmp/subscribe_output.txt && echo "subscribe entry point executed"'
           docker run --rm ${{ steps.get_tag.outputs.image }} bash -c 'source /home/client/.venv/bin/activate && timeout 2 update_inuse_status > /tmp/update_output.txt 2>&1 || true && test -s /tmp/update_output.txt && echo "update_inuse_status entry point executed"'
+
+      - name: Verify custom-startup.sh execution
+        run: |
+          echo "Testing custom-startup.sh execution..."
+          CUSTOM_STARTUP_SCRIPT_CONTENT=$(cat <<'EOF'
+          #!/bin/bash
+          echo "CUSTOM_STARTUP_SCRIPT_EXECUTED_SUCCESSFULLY"
+          sudo apt-get update -y
+          sudo apt-get install -y jq
+          echo "JQ_VERSION: $(jq --version)"
```

Comment:
could keep a few different scripts in the tests folder maybe in a test_data folder and have a separate CI to try different typical test scripts. We could keep this CI shorter but we don't need to necessarily since it has stages and we already have the shorter CI testing.